src/pages/[platform]/start/mobile-support/index.mdx

```diff
@@ -258,257 +223,543 @@ export const data = defineData({
-)
+<video autoPlay={true} muted={true} loop={true} width="100%" playsInline={true}>
+  <source src="/images/gen2/getting-started/flutter/flutter-getting-started-2.mp4" />
+</video>
+
+You can terminate the sandbox environment now to clean up the project.
+
+### Publishing changes to cloud
+
+For publishing the changes to cloud, you need to create a remote git repository. For a detailed guide, you can follow the link [here](/[platform]/start/quickstart/#create-remote-git-repository).
+
+</Block>
+
+<Block name="React Native (Expo)">
+
+For this tutorial you will be using [Expo](https://expo.dev/)'s TypeScript template to create your application.
+
+  <Callout>
+
+Amplify now requires native modules not available through the Expo SDK. As a result, Expo Go is no longer supported but you should still be able to use Expo. [Learn more about dropping support for Expo Go in Amplify v6](/react-native/build-a-backend/troubleshooting/migrate-from-javascript-v5-to-v6/#dropping-support-for-expo-go).
+
+</Callout>
+
+```sh
+npx create-expo-app my_amplify_app -t expo-template-blank-typescript
+cd my_amplify_app
 ```
 
-For deleting add a long click behavior with the `Modifier.combinedClickable` modifier. To add it, update the Row composable call in the `TodoScreen` composable function with the following code:
+<Callout warning>
+For calling native libraries and platform dependencies, you need to have run the prebuild command for generating the folders for depending platforms. 
 
-```kotlin
-Row(
-    verticalAlignment = Alignment.CenterVertically,
-    modifier = Modifier.combinedClickable(
-        onClick = {
-            val newTodo = todo.copyOfBuilder().isDone(!todo.isDone).build()
-            Amplify.API.mutate(
-                ModelMutation.update(newTodo),
-                {
-                    Log.i("MyAmplifyApp", "Updated Todo with id: ${todo.id}")
-                    onItemUpdated(newTodo)
-                },
-                { Log.e("MyAmplifyApp", "Update failed") }
-            )
-        },
-        onLongClick = {
-            Amplify.API.mutate(
-                ModelMutation.delete(todo),
-                {
-                    Log.i("MyAmplifyApp", "Deleted Todo with id: ${todo.id}")
-                    onItemDeleted(todo)
-                },
-                { Log.e("MyAmplifyApp", "Delete failed") }
-            )
-        },
-    )
-)
+```sh
+npx expo prebuild
 ```
+</Callout>
 
-With the click, we update the checkbox but with the long click we remove it. Now if you run the application you should see the following flow.
+### Create Amplify Project
 
-<video autoPlay={true} muted={true} loop={true} width="40%" playsInline={true}>
-  <source src="/images/gen2/getting-started/android/android-getting-started-2.mp4" />
+The easiest way to get started with AWS Amplify is through npm with `create-amplify` command. You can run it from your base project directory.
+
+```bash
+npm create amplify@beta
```

Comment:
For release, this won't be `@beta` I don't think.

Reply:
This is targeting to improve support for current docs.